From 61a1aea7c7cb40de071e202cfaa31fa2c1fca8ba Mon Sep 17 00:00:00 2001
Message-Id: <61a1aea7c7cb40de071e202cfaa31fa2c1fca8ba.1399488296.git.tomas.winkler@intel.com>
In-Reply-To: <4fcbc99b7565f915bea58e14b5e6f089bf9abf16.1399488296.git.tomas.winkler@intel.com>
References: <4fcbc99b7565f915bea58e14b5e6f089bf9abf16.1399488296.git.tomas.winkler@intel.com>
From: Tomas Winkler <tomas.winkler@intel.com>
Date: Tue, 18 Mar 2014 22:52:07 +0200
Subject: [char-misc-next 13/16] mei: me: bump hbm version to 1.1 to support
 power gating

Communicate hbm version 1.1 to firmware to tell that we
now support power gating isolation protocol

Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/misc/mei/hw.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/mei/hw.h b/drivers/misc/mei/hw.h
index 9bad9cf..dd448e5 100644
--- a/drivers/misc/mei/hw.h
+++ b/drivers/misc/mei/hw.h
@@ -37,7 +37,7 @@
 /*
  * MEI Version
  */
-#define HBM_MINOR_VERSION                   0
+#define HBM_MINOR_VERSION                   1
 #define HBM_MAJOR_VERSION                   1
 
 /*
-- 
1.9.0

